test/e2e/page-objects/pages/home/activity-list.ts

```diff
@@ -0,0 +1,129 @@
+import { strict as assert } from 'assert';
+import { Driver } from '../../../webdriver/driver';
+
+class ActivityListPage {
+  private readonly driver: Driver;
+
+  private readonly completedTransactions = '[data-testid="activity-list-item"]';
+
+  private readonly confirmedTransactions = {
```

Comment:
@chloeYue - We should avoid using CSS and text format, as it would not fail. I have shared a screenshot showing that entering incorrect text format still results in a successful test. 

<img width="1456" alt="Screenshot 2024-12-02 at 9 57 42 AM" src="https://github.com/user-attachments/assets/1c91c1a8-1789-4296-a0ac-26489c28c7df">

Reply:
I looked at the code and we may need to add the data-testid locator for this component. 

Since this PR is to break down the home page into multiple pages, you could address this in another PR and I will leave it up to you to decide.